Jenkinsfile

```diff
@@ -38,7 +38,7 @@
  */
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value="pipeline-lib@your_branch") _
+@Library(value="pipeline-lib@pipeline-pr-test-rpms") _
```

Comment:
Presumably this won't actually me landed until after the pipeline-lib changes are landed?

Reply:
That's right.  This is exactly the kind of thing that commented out `Library()` call is there for.